packages/view-types/feature-list/src/FeatureListOptions.js

```diff
@@ -35,16 +37,17 @@ export default function FeatureListOptions(props) {
     <OptionsContainer>
       {children}
       <TableRow>
-        <TableCell className={classes.labelCell} htmlFor="feature-list-sort-option-select">
-          Sort Ordering
+        <TableCell className={classes.labelCell} variant="head" scope="row">
+          <label htmlFor={['feature-list-sort-option', featureListId].join('-')}>Sort Ordering</label>
```

Comment:
I think it might be a bit clearer to use a template string than an array, e.g.:

```suggestion
          <label htmlFor={`feature-list-sort-option-${featureListId}`}>Sort Ordering</label>
```